common/src/main/java/com/wynntils/utils/objects/Location.java

```diff
@@ -93,7 +93,6 @@ public final BlockPos toBlockPos() {
     }
 
     public Location clone() throws CloneNotSupportedException {
-        Location clone = (Location) super.clone();
```

Comment:
I think you are forgetting that super.clone() can have side effects. You cannot just remove it. I don't remember the details about clone (it was a bad design decision anyway), but I'm pretty sure you are required to call super.clone.

Reply:
I'm not sure that the comments adds anything of value? All non-expression statements are in place for their side effects, that's not anything you need to state.